CHANGELOG.md

```diff
@@ -15,6 +15,17 @@ This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.htm
     _Security_ in case of vulnerabilities.
  -->
 
+## [0.19.0](TODO)
```

Comment:
Saving this for the final commit.

Reply:
I recommend to merge main and leave the Unreleased header as is

Reply:
Just to confirm, it's good as is? / you or other team members will update this before the next release?

Reply:
Revert the changes to the original commit :D 

The workflow to update the CHANGELOG goes like so:
1. For each PR, add the functionality under the appropriate section with a link to your PR
2. When it comes to release, we use a bi-weekly cadence with this [CI job running](https://github.com/o1-labs/o1js/blob/main/.github/workflows/release.yml) which will automate the commit range for you. 
3. If you want to do it manually, you can also use this script [here](https://github.com/o1-labs/o1js/blob/main/update-changelog.sh)

That is to say, in normal development, you don't have to worry about it. It will be handled by CI and reviewed when a release PR is made. Like [this one for example](https://github.com/o1-labs/o1js/pull/1510). It would be nice to document this in our README-dev since it's not super clear at this point.